integration_tests/test_rough_align.py

```diff
@@ -262,11 +284,15 @@ def test_apply_rough_alignment_transform(render, montage_stack, test_do_rough_al
         mod.run()
 
     mod = ApplyRoughAlignmentTransform(input_data=ex3, args=[])
-    mod.run()
+    with pytest.raises(RenderModuleException):
```

Comment:
I don't see what this is testing -- I think this `RenderModuleException` is actually a result of the base example filling in the new z value of 251, which causes a `RenderError` in getting tilespecs and is caught as a `RenderModuleException` when validating that the processes completed successfully.

Reply:
Also, it would probably make sense to test the `map_z_start=True` capabilities in something other than the error case.